homeassistant/components/workday/repairs.py

```diff
@@ -106,6 +106,61 @@ async def async_step_province(
         )
 
 
+class HolidayFixFlow(RepairsFlow):
+    """Handler for an issue fixing flow."""
+
+    def __init__(
+        self, entry: ConfigEntry, country: str | None, named_holiday: str
+    ) -> None:
+        """Create flow."""
+        self.entry = entry
+        self.country: str | None = country
+        self.named_holiday: str = named_holiday
+        super().__init__()
+
+    async def async_step_init(
+        self, user_input: dict[str, str] | None = None
+    ) -> data_entry_flow.FlowResult:
+        """Handle the first step of a fix flow."""
+        return await self.async_step_named_holiday()
+
+    async def async_step_named_holiday(
+        self, user_input: dict[str, Any] | None = None
+    ) -> data_entry_flow.FlowResult:
+        """Handle the options step of a fix flow."""
+        if user_input:
+            options = dict(self.entry.options)
+            new_options = {**options, **user_input}
+            self.hass.config_entries.async_update_entry(self.entry, options=new_options)
+            await self.hass.config_entries.async_reload(self.entry.entry_id)
+            return self.async_create_entry(data={})
+
+        new_schema = self.add_suggested_values_to_schema(
+            vol.Schema(
+                {
+                    vol.Optional(CONF_REMOVE_HOLIDAYS, default=[]): SelectSelector(
+                        SelectSelectorConfig(
+                            options=[],
+                            multiple=True,
+                            custom_value=True,
+                            mode=SelectSelectorMode.DROPDOWN,
+                        )
+                    ),
+                }
+            ),
+            user_input or self.entry.options,
```

Comment:
```suggestion
            self.entry.options,
```

I would also remove the invalid holiday name from the suggested values